NoStringEvaluating/Contract/INoStringEvaluator.cs

```diff
@@ -5,416 +5,417 @@
 using NoStringEvaluating.Models.Values;
 using NoStringEvaluating.Nodes.Common;
 
-namespace NoStringEvaluating.Contract;
-
-/// <summary>
-/// Expression evaluator
-/// </summary>
-public interface INoStringEvaluator
+namespace NoStringEvaluating.Contract
```

Comment:
Don't change design for namespaces, please, fix in all files. Have you rebased successfully?